<launch>
	<group ns="hostess_skeleton_tracker">
		<param name="camera_frame_id" type="str" value="camera_depth_frame" />
		<param name="user_to_track" type="str" value="00000004" />
		<param name="skeleton_to_track" type="int" value="0" />

		<node pkg="hostess_skeleton_tracker" type="hostess_skeleton_kalman" name="hostess_skeleton_kalman" output="screen" />
		<node pkg="hostess_skeleton_tracker" type="twist_mediator" name="twist_mediator" output="screen" />
	</group>
</launch>
